code/datums/mood_events/generic_negative_events.dm

```diff
@@ -311,6 +311,25 @@
 	mood_change = -3
 	timeout = 90 SECONDS
 
+/datum/mood_event/tail_lost
+	description = "<span class='boldwarning'>My tail!! Why?!</span>\n"
+	mood_change = -8
+	timeout = 10 MINUTES
+
+/datum/mood_event/tail_balance_lost
```

Comment:
Yeah if you get this through I'm definitely going to PR a trip chance to this moodlet.

Reply:
if you go that route you might want to roll this into a status effect